uint8 GREEN = 3
uint8 YELLOW = 2
uint8 RED = 1

time timestamp
uint8 traffic_light
